src/morph/src/Response.re

```diff
@@ -60,86 +62,81 @@ let set_status = (status: Status.t) => {
   result_map(res => {...res, status});
 };
 
-let set_body = (body: body) => {
+let set_body = body => {
   result_map(res => {...res, body});
 };
 
-let ok = (res: t) => {
+let ok = res => {
   add_header(("Content-length", "2"), res)
   |> set_status(`OK)
-  |> set_body(`String("ok"))
-  |> Lwt.return;
+  |> set_body(`String("ok"));
```

Comment:
This  and the following `Lwt.return` was removed when I tried to remove Lwt from the core, maybe I should add it back to not clutter the user code?